StreamChatUITestsApp/StreamChat/StreamChatWrapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import StreamChatUI
+import UIKit
+
+final class StreamChatWrapper {
```

Comment:
Nice wrapper 👍  Wonder if we can/should add offline mode support for the UI tests?

Reply:
That part is planned in our epic (CIS-1509) - see tasks.

This PR provides basic testing infra in place that already allows us (not only me & Alex) to build a robust test suite for the most common cases.